src/os/bluestore/BlueStore.cc

```diff
@@ -12107,21 +12113,6 @@ void BlueStore::_get_statfs_overall(struct store_statfs_t *buf)
       bluefs->get_used()
       - buf->omap_allocated;
   }
-
-  ExtBlkDevState ebd_state;
-  int rc = bdev->get_ebd_state(ebd_state);
-  if (rc == 0) {
-    buf->total += ebd_state.get_physical_total();
-
-    // we are limited by both the size of the virtual device and the
-    // underlying physical device.
-    bfree = std::min(bfree, ebd_state.get_physical_avail());
-
-    buf->allocated = ebd_state.get_physical_total() - ebd_state.get_physical_avail();;
```

Comment:
Interesting, this `buf->allocated` was overwritten in `BlueStore::statfs()`.

Reply:
true, but apparently nobody cared...